ui-kit/GroupCard/GroupCard.styles.js

```diff
@@ -42,6 +40,11 @@ const GradientBackground = styled.div`
   position: relative;
 `;
 
+const GroupCard = styled(Card)`
```

Comment:
Instead of doing `styled(Card)`, you can also just do `as={Card}` on `<Styled>`. But this also works fine, too 👍

Reply:
So I would add `as={Card}` to `<Styled>` inside of the `GroupCard.js`?

Reply:
Yes, you can do that, but it's probably better to just leave this how it is 👌

Reply:
The only other thing I'd say we change is to move `GroupCard` to the top of this file since that's the default component being exported.